packages/core/src/html/NodeProcessor.ts

```diff
@@ -28,11 +32,7 @@ const { processInclude, processPanelSrc, processPopoverSrc } = require('./includ
 const linkProcessor = require('./linkProcessor');
 const { highlightCodeBlock, setCodeLineNumbers } = require('./codeblockProcessor');
 const { setHeadingId, assignPanelId } = require('./headerProcessor');
-const { MarkdownProcessor } = require('./MarkdownProcessor');
 const { FootnoteProcessor } = require('./FootnoteProcessor');
-const { MdAttributeRenderer } = require('./MdAttributeRenderer');
-const { shiftSlotNodeDeeper, transformOldSlotSyntax, renameSlot } = require('./vueSlotSyntaxProcessor');
-const { warnConflictingAtributesMap, warnDeprecatedAtributesMap } = require('./warnings');
 const { processScriptAndStyleTag } = require('./scriptAndStyleTagProcessor');
```

Comment:
Could this also be an ES6 import since scriptAndStyleTagProcessor has been migrated?

Reply:
Sure, I also made a minor modification in the `processScriptAndStyleTag` func so that TS won't complain.